### ISSUE TITLE
[Bug]: Ubuntu Samba folder sharing under guest access still requires a password. 

### ISSUE THREAD

**Issue by Pentaphon** [nova-video-player/aos-AVP#568]
### Problem description

Ubuntu Samba folder sharing under guest access still requires a password. 

### Steps to reproduce the issue

1) Follow a tutorial such as https://itsfoss.com/share-folders-local-network-ubuntu-windows/ (all SMB settings are default)
2) Set up a folder to share over the LAN as the screenshot shows:
![PUFnU](https://user-images.githubusercontent.com/49374058/153962728-7ffa0872-2b94-42f7-a524-948a129548bc.png)
3) Try to connect to shared folder under SMB. The shared folder shows up fine but when I try to access the folder, it asks me for a password. 
 

### Expected behavior

I should be able to connect to the SMB shared folder without needing a password. 

### Your phone/tablet/androidTV model

Onn 4K 

### Operating system version

Android 10

### Application version and app store

6.0.43

### Additional system information

N/A

### Debug logs

This is easily reproducible so none are applicable.

**Comment by okan35** [comment]
I dont think this is a bug for nova. You need to create settings for that and it has always been a hell to do that on any linus os.

**Comment by Pentaphon** [comment]
> I dont think this is a bug for nova. You need to create settings for that and it has always been a hell to do that on any linus os.

I tried looking at the Wiki and there's nothing about this. You may be right, but I would think that those settings in the GUI would be enough.

**Comment by rahaaatul** [comment]
Off topic but do you know what to put in domain in SMB credentials?

**Comment by Pentaphon** [comment]
> Off topic but do you know what to put in domain in SMB credentials?

I'm not sure. We need a tutorial on how to connect to Linux shares. 

@courville do you connect to Ubuntu or other similar Linux samba shares? Any tips on this?

**Comment by courville** [comment]
I have just enabled SMB3 in latest jcifs-ng on an experimental release that could solve the issue. Please test and report nova v6.0.68 available here https://github.com/nova-video-player/aos-AVP/releases/tag/v6.0.68
